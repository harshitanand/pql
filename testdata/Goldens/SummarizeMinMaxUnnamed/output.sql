SELECT min("LineCount") AS "min(LineCount)", max("LineCount") AS "max(LineCount)" FROM "SourceFiles";
