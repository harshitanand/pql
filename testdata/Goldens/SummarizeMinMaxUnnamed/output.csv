min(LineCount),max(LineCount)
24,1108
